### ISSUE TITLE
FxCop extension authored incorrectly

### ISSUE THREAD

**Issue by jmarolf** [dotnet/roslyn-analyzers#1253]
the extension available [here](https://marketplace.visualstudio.com/items?itemName=VisualStudioPlatformTeam.MicrosoftCodeAnalysis2017) is not authored correctly.

The VSIX manifest is allows this extension to be installed on any version of VS that is 2017 RTW or newer.
```XML
<Installation>
  <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.0,]" />
  <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[15.0,]" />
</Installation>
```

following the [docs](https://docs.microsoft.com/en-us/visualstudio/extensibility/vsix-extension-schema-2-0-reference) this should read:
```XML
<Installation>
  <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.3.0,]" />
  <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[15.3.0,]" />
</Installation>
```

Or perhaps even a more specific version

**Comment by sharwell** [comment]
> following the docs this should read:

Are we sure the ~~gallery~~ marketplace will work in this scenario?

:bulb: The `Microsoft.VisualStudio.Pro` line is unnecessary unless you are planning to target Visual Studio 2013 (or earlier) scenarios.

**Comment by jmarolf** [comment]
@sharwell good point. `Microsoft.VisualStudio.Pro` is unnecessary. 

>Are we sure the gallery marketplace will work in this scenario?

Is your question whether the VS marketplace restricts VSIX downloads?  It does not.  You can filter to just show you things for VS 2017 but it does not get any more specific than that.

Correctly authoring the extension will warn users when they try to install the extension though, which is what you want instead of VS simply failing to load it.

**Comment by sharwell** [comment]
> Is your question whether the VS marketplace restricts VSIX downloads?

No, I was more concerned with a case where it thinks Visual Studio 2017 is "15.0" and doesn't show it at all for 15.3.

**Comment by jmarolf** [comment]
@sharwell  that is not how the Visual Studio Marketplace filtering works.  Visual  Studio 2017 is 15.*

I do not believe any work was done on the marketplace side to account for individual 15.x releases

**Comment by mavasani** [comment]
Issue specific to our setup projects for VSIX

**Comment by mavasani** [comment]
Blocked on knowing the 15.5 version
